exporter/zipkinexporter/zipkin.go

```diff
@@ -68,15 +69,20 @@ func newZipkinExporter(finalEndpointURI, defaultServiceName string, uploadPeriod
 	return zle, nil
 }
 
-func lookupAttribute(node *commonpb.Node, key string) string {
-	if node == nil {
-		return ""
-	}
-	return node.Attributes[key]
-}
-
-func zipkinEndpointFromNode(node *commonpb.Node, serviceName string, endpointType zipkinDirection) *zipkinmodel.Endpoint {
-	if node == nil {
+// zipkinEndpointFromAttributes attempts to extrace zipkin endpoint information
```

Comment:
suggestions:

1. At a minimum: s/attempts to extrace/extracts/ (all code attempts to do something).
2. What happens when > 3 attribute keys are made redundant?

Reply:
1. ✅
2. The comment is not future proof. If we change the function we should review it.